gradoop-examples/gradoop-examples-temporal/src/main/java/org/gradoop/examples/aggregation/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Examples related to the temporal aggregation operator
```

Comment:
```suggestion
 * Examples related to the temporal aggregation operator.
```
Also unsure about the naming here, since this is not a separate operator, its just the regular aggregation operator with some additional API functions